workflow/applications/gfs_cycled.py

```diff
@@ -83,12 +83,9 @@ def _get_app_configs(self, run):
                 configs += ['atmensanlinit', 'atmensanlobs', 'atmensanlsol',
                             'atmensanlletkf', 'atmensanlfv3inc', 'atmensanlfinal']
             else:
-                configs += ['eobs', 'eomg', 'ediag', 'eupd']
+                configs += ['eobs', 'ediag', 'eupd']
 
-            configs += ['ecen', 'esfc', 'efcs', 'echgres', 'epos', 'earc_vrfy']
-
-            if options['do_archtar']:
-                configs += ['earc_tars']
+            configs += ['ecen', 'esfc', 'efcs', 'echgres', 'epos', 'earc']
```

Comment:
I think this should be reverted.

```suggestion
            configs += ['ecen', 'esfc', 'efcs', 'echgres', 'epos', 'earc_vrfy']

            if options['do_archtar']:
                configs += ['earc_tars']
```

Reply:
Done